src/io.d

```diff
@@ -68,11 +84,16 @@ struct Input
 
     Position position() const pure @safe
     {
-        import std.algorithm : find;
+        if (lsSupport) {    
```

Comment:
I like the idea that `line` is set to 0 and is never incremented in case of `useOffset`.
Then no change is required here.

Reply:
I'm note sure in that. The special circumstance of `line==0` implying offset handling is too implicit here. When reading it later, it will be hard to figure out. The passing of zero makes it more explicit, no further documentation hints needed.
Furthermore, we could avoid to `find` the text for position marking, as it is useless in case of offset reporting.